src/tests/jsonLogic.test.js

```diff
@@ -82,6 +89,82 @@ describe('cross-value validations', () => {
     });
   });
 
+  describe('Incorrectly written schemas', () => {
+    beforeEach(() => {
+      jest.spyOn(console, 'error').mockImplementation(() => {});
+    });
+
+    afterEach(() => {
+      console.error.mockRestore();
+    });
+
+    it('Should throw when theres a missing rule', () => {
+      createHeadlessForm(schemaWithMissingRule, { strictInputType: false });
+      expect(console.error).toHaveBeenCalledWith(
+        'JSON Schema invalid!',
+        Error('Missing rule for validation with id of: "a_greater_than_ten".')
+      );
+    });
+
+    it('Should throw when theres a missing computed value', () => {
+      createHeadlessForm(schemaWithMissingComputedValue, { strictInputType: false });
+      expect(console.error).toHaveBeenCalledWith(
+        'JSON Schema invalid!',
+        Error('Missing rule for computedValue with id of: "a_plus_ten".')
+      );
+    });
+
+    it('Should throw when theres an inline computed ruleset with no value.', () => {
+      createHeadlessForm(schemaWithMissingValueInlineRule, { strictInputType: false });
+      expect(console.error).toHaveBeenCalledWith(
+        'JSON Schema invalid!',
+        Error('Cannot define multiple rules without a template string with key `value`.')
+      );
+    });
+
+    it('On x-jsf-logic-computedAttrs, error if theres a value that does not exist.', () => {
```

Comment:
nit: Did you write these tests on a different day? Their description are not consistent compared to the ones above 😅  More similar: "Should throw on x-jsf-logic-computedAttrs, if there's a value that does not exist."

Or may personal favorite which cuts "should", and uses "given" instead of "if": "On x-jsf-logic-computedAttrs, throw error given a value that doesn't exist"

Reply:
i tried to clean this up with https://github.com/remoteoss/json-schema-form/pull/38/commits/81402ed910a066a3d3ba2a7c1dd58f808548deed. Another day of https://www.youtube.com/watch?v=JXFGy10b7Js, LMK if this could be better

Reply:
there you go, green card eheh